src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -1040,19 +1040,19 @@ private static bool TryGetArg0AsDsInfo(CallNode callNode, TexlBinding binding, o
             return result;
         }
 
-        protected bool SetErrorForMismatchedColumns(DType expectedType, DType actualType, TexlNode errorArg, IErrorContainer errors, Features features)
+        internal static bool SetErrorForMismatchedColumns(DType expectedType, DType actualType, TexlNode errorArg, IErrorContainer errors, Features features, bool requireAllParamColumns)
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

We should revert this to `protected` when the definition for the Collect/Remove functions are moved to the MS.PFx.Core assembly. Please create an issue to track that.

Reply:
reverted it to `protected` instead chaining it to static method, so it stays PA non breaking